casepro/backend/junebug.py

```diff
@@ -40,6 +46,73 @@ def get_addresses(self, uuid):
         return (
             a['address'] for a in addresses if a.get('address') is not None)
 
+    def get_identities(self, **params):
+        '''Get the list of identities filtered by the given kwargs.'''
+        url = '%s/api/v1/identities/search/?' % self.base_url
+
+        identities = self.get_paginated_response(
+            url, params=params)
+
+        # Users who opt to be forgotten from the system have their details
+        # stored as 'removed'
+        return (
+            IdentityStoreContact(i) for i in identities if
+            i.get('details').get('name') is not "removed"
```

Comment:
This isn't quite right. See https://github.com/praekelt/seed-identity-store/blob/develop/identities/models.py#L76-L86

If someone's requested forget me, then when you search for them by address_type/address you won't find them. If you happen to access them directly using `/identity/uuid/`you'll see their `details["addresses"] == {}`. Any optional details ("name" is optional) would be `redacted` not `removed`.